packages/webpack/src/plugins/nx-webpack-plugin/lib/normalize-options.ts

```diff
@@ -51,14 +51,16 @@ export function normalizeOptions(
     }
     Object.assign(
       combinedPluginAndMaybeExecutorOptions,
-      buildTargetOptions,
-      options // plugin options take precedence
+      options,
+      // executor options take precedence (especially for overriding with CLI args)
+      buildTargetOptions
     );
   } else {
     Object.assign(
       combinedPluginAndMaybeExecutorOptions,
-      originalTargetOptions,
-      options // plugin options take precedence
+      options,
+      // executor options take precedence (especially for overriding with CLI args)
+      originalTargetOptions
```

Comment:
The order here was previously set wrong. The executor options should override plugin options, since users can pass CLI args, or have overrides in `configurations`.